<ui version="4.0" >
 <class>addBuddyScreen</class>
 <widget class="QDialog" name="addBuddyScreen" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>208</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>250</width>
    <height>208</height>
   </size>
  </property>
  <property name="font" >
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/slm_resources.qrc" >
    <normaloff>:/icons/SLM_Logo</normaloff>:/icons/SLM_Logo</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Enter Buddy Alias</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <widget class="QLabel" name="L_buddyAliasLabel" >
          <property name="text" >
           <string>Alias</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="L_buddyAliasLine" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Enter Host Name or IP Address of the Buddy</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="QLabel" name="L_hostNameLabel" >
          <property name="text" >
           <string>Host Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="L_buddyHostNameLine" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLabel" name="L_buddyIPAddressLabel" >
          <property name="text" >
           <string>IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="L_buddyIPAddressLine" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="L_addBuddyButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="L_cancelAddBuddyButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/slm_resources.qrc" />
 </resources>
 <connections/>
</ui>
